augur/clades.py

```diff
@@ -0,0 +1,89 @@
+import os, sys
+import numpy as np
+from Bio import SeqIO, SeqFeature, Seq, SeqRecord, Phylo
+from .utils import read_node_data, write_json
+from collections import defaultdict
+
+def read_in_clade_definitions(clade_file):
+    '''
+    Reads in tab-seperated file that defines clades by amino-acid.

```

Comment:
This docstring made me think clades couldn't be defined by nucleotides, but seeing the real TB clade definitions clarified this. Could you update the docstring comment and example to mention nucleotides, too?